src/visitor.rs

```diff
@@ -90,17 +90,20 @@ impl<'a> FmtVisitor<'a> {
         }
     }
 
-    pub fn visit_block(&mut self, b: &ast::Block, inner_attrs: Option<&[ast::Attribute]>) {
+    pub fn visit_block(
+        &mut self,
+        b: &ast::Block,
+        inner_attrs: Option<&[ast::Attribute]>,
+        is_dummy: bool,
```

Comment:
COuld we call `is_dummy` `has_braces` or something similar? It's not obvious to me what `is_dummy` means.

Reply:
Thanks for the review!

`is_dummy` is `true` when we manually construct a block for the body of closure. Since the block is not present in the original code I called it dummy.
I agree that using `has_braces` is better, it is more natural to use a flag that is `true` most of the time and `false` in a narrow case. And the name is clearer.